### DISCUSSION TITLE
KPN IPTV freezes and gives: foutcode: STB-nmc-400

### DISCUSSION THREAD

**Discussion by EelcoHoff** [fabianishere/udm-iptv#272] [Q&A]
Hi All,

After my UDM Pro was updated to the newest release the IPTV from KPN Freezes after a few seconds starts freezes again and then gives error STB-nmc-400.

version udm pro:
Network  7.4.162
UniFi OS  UDM Pro v3.1.15

The diagnostics returns the following:
=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br4
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
38: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.214.80.57/21 brd 10.214.87.255 scope global iptv
       valid_lft forever preferred_lft forever
10.214.80.0/21 proto kernel scope link src 10.214.80.57
213.75.112.0/21 via 10.214.80.1 metric 238
=== Service Logs ===
Aug 27 12:02:30 UDMPRO-Hoff udm-iptvd[79690]: RECV V2 member report   from 192.168.4.1     to 233.89.188.1
Aug 27 12:02:30 UDMPRO-Hoff udm-iptvd[79690]: The IGMP message was from myself. Ignoring.
Aug 27 12:02:30 UDMPRO-Hoff udm-iptvd[79690]: The source address 192.168.4.1 for group 233.89.188.1 is from downstream VIF[0]. Ignoring.
Aug 27 12:02:36 UDMPRO-Hoff udm-iptvd[79690]: RECV Membership query   from 192.168.4.1     to 224.0.0.1
Aug 27 12:02:37 UDMPRO-Hoff udm-iptvd[79690]: RECV V2 member report   from 192.168.4.1     to 224.0.0.22
Aug 27 12:02:37 UDMPRO-Hoff udm-iptvd[79690]: The IGMP message was from myself. Ignoring.
Aug 27 12:02:38 UDMPRO-Hoff udm-iptvd[79690]: RECV V2 member report   from 192.168.4.1     to 224.0.0.251
Aug 27 12:02:38 UDMPRO-Hoff udm-iptvd[79690]: The IGMP message was from myself. Ignoring.
Aug 27 12:02:39 UDMPRO-Hoff udm-iptvd[79690]: RECV V2 member report   from 192.168.4.1     to 224.0.0.2
Aug 27 12:02:39 UDMPRO-Hoff udm-iptvd[79690]: The IGMP message was from myself. Ignoring.

Did I do somthing wrong anybody an idee of what i could try?

Regards, Eelco

**Comment by EelcoHoff** [comment]
Never mind, got it working again by switching to br0.
Think i messed up the vlan someway.

**Comment by hkerssies** [comment]
I am not so lucky,
Still not getting it working. Or it freesez or get the STB-nmc-400.

I use a UDM SE firmware 3.2.7 and also on 3.1.16 have a internal VLAN4 for iptv. Behind the router is a USW-Lite-16-PoE with igmp snooping.

```
=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br4
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
30: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.227.146.158/22 brd 10.227.147.255 scope global iptv
       valid_lft forever preferred_lft forever
10.227.144.0/22 proto kernel scope link src 10.227.146.158 
213.75.112.0/21 via 10.227.144.1 metric 230 
=== Service Logs ===
Dec 15 18:42:20 UDM-SE-HVS6 udm-iptvd[104444]: Obtaining IP address for VLAN interface
Dec 15 18:42:20 UDM-SE-HVS6 udm-iptvd[104450]: udhcpc: started, v1.30.1
Dec 15 18:42:20 UDM-SE-HVS6 udm-iptvd[104450]: udhcpc: sending discover
Dec 15 18:42:20 UDM-SE-HVS6 udm-iptvd[104450]: udhcpc: sending select for 10.227.146.158
Dec 15 18:42:20 UDM-SE-HVS6 udm-iptvd[104450]: udhcpc: lease of 10.227.146.158 obtained, lease time 3600
Dec 15 18:42:20 UDM-SE-HVS6 udm-iptvd[104444]: NATing IPTV network ranges (if necessary)
Dec 15 18:42:20 UDM-SE-HVS6 udm-iptvd[104444]: Creating static routes (if necessary)
Dec 15 18:42:20 UDM-SE-HVS6 udm-iptvd[104444]: Setting up IGMP Proxy
Dec 15 18:42:20 UDM-SE-HVS6 udm-iptvd[104444]: Using improxy...
Dec 15 18:42:20 UDM-SE-HVS6 udm-iptvd[104444]: Starting IGMP Proxy
Dec 15 20:08:23 UDM-SE-HVS6 udm-iptvd[3040]: DEBUG[main@655]: timeout
Dec 15 20:08:23 UDM-SE-HVS6 udm-iptvd[3040]: DEBUG[main@655]: timeout
Dec 15 20:08:23 UDM-SE-HVS6 udm-iptvd[3040]: DEBUG[main@655]: timeout
Dec 15 20:08:23 UDM-SE-HVS6 udm-iptvd[3040]: DEBUG[main@655]: timeout
Dec 15 20:08:23 UDM-SE-HVS6 udm-iptvd[3040]: DEBUG[main@655]: timeout
Dec 15 20:08:23 UDM-SE-HVS6 udm-iptvd[3040]: DEBUG[main@655]: timeout
Dec 15 20:08:23 UDM-SE-HVS6 udm-iptvd[3040]: DEBUG[main@655]: timeout

```


USW-Lite-16-PoE:
```
ip igmp snooping
no ip igmp snooping report-suppression
ip igmp snooping version 3
ip igmp snooping vlan 1,4
ip igmp snooping vlan 1 querier version 2
ip igmp snooping vlan 4 querier version 2
ipv6 mld snooping
no ipv6 mld snooping report-suppression
ipv6 mld snooping vlan 1,4 

interface gi15
 switchport mode hybrid
 switchport hybrid pvid 4
 switchport hybrid allowed vlan add 1 tagged
 switchport hybrid allowed vlan add 4 untagged
 switchport default-vlan tagged
 authentication guest-vlan
 storm-control unknown-multicast
 storm-control unknown-multicast level 100
 storm-control broadcast
 storm-control broadcast level 100
 storm-control unknown-unicast
 storm-control unknown-unicast level 100
